message/message.go

```diff
@@ -218,3 +218,21 @@ func (nc *NetworkCurve) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+const PreAuthEndpoint = "/v1/enduser-auth/preauth"
```

Comment:
```suggestion
const PreAuthEndpoint = "/v1/endpoint-auth/preauth"
```

changed shortly after you wrote this PR